examples/basic/variables.tf

```diff
@@ -2,32 +2,35 @@
 # Input variables
 ########################################################################################################################
 
-variable "ibmcloud_api_key" {
+variable "resource_group" {
   type        = string
-  description = "The IBM Cloud API Key"
-  sensitive   = true
+  description = "An existing resource group name to use for this example. If not specified, a new resource group is created."
+  default     = null
 }
 
-variable "region" {
+variable "ibmcloud_api_key" {
+  description = "Used with the Terraform IBM-Cloud/ibm provider"
+  sensitive   = true
   type        = string
-  description = "Region to provision all resources created by this example"
-  default     = "us-south"
 }
 
-variable "prefix" {
+variable "location" {
```

Comment:
this is the correct terminology for this service, but do we want to keep region for consistency across modules? @ocofaigh

Reply:
Agree - use `region`